### DISCUSSION TITLE
Pl2 duration and fan speed percentage

### DISCUSSION THREAD

**Discussion by WILLYWZ** [seerge/g-helper#3221] [General]
It occurred to me that in cpu section of the gheper PL2 is said to be CPU 2 min boost? 
As far as i know, PL2 duration is **usually 56 seconds** by default and in most cases **does not excced 100 seconds**.

also the cpu fan speed in the main UI is different , especiality **after manual calibration**.

Just FYI. All those are not really a big deal.

**Comment by seerge** [comment]
@WILLYWZ hello, Asus says that this is 2 minutes in [their article about Armoury](https://rog.asus.com/articles/guides/how-to-tweak-your-rog-laptops-performance-with-armoury-crates-manual-mode/). Anyway this is just a label explaining people what it is :)

> On Intel machines, PL2 denotes the highest power target your CPU is capable of, though it can only boost this high for a maximum of **2 minutes**. After that time period, it will ramp down to its PL1 value, which is the maximum power target it can sustain indefinitely. 

Calibration does not change the UI, it only measures max fan speed in RPM and adjusts Y-axis on fan charts with new values. Fan speed % on main window is also calculated based on currently reported RPM and measured max fan speed.

**Reply by WILLYWZ** [reply]
Then I guess Intel have different a standard (based on XTU) than what Asus have. 
Also let me give an example of the fan speed display. 
![image](https://github.com/user-attachments/assets/355be993-e925-4610-a055-83f888f320dc)
![image](https://github.com/user-attachments/assets/704bb107-4628-47c8-bf84-abaf87c76ef7)
![image](https://github.com/user-attachments/assets/9ccde88d-6959-49f7-a4d2-11e7558345f7)

**Reply by WILLYWZ** [reply]
The rpm and percentage isn't aligned. 2400rpm on cpu fan is displayed as 36% but it is in fact less than 20% on the scale (after calibration). 
Same thing to all other fans. GPU is 2600rpm display as 39% but the scale said its close to 20%. 
2600/0.2 to a insane 13000 rpm where fans top off at 6600.
So i think the scale itself in fan curve have incorrect values for fan speed (and its corresponding percentage). 
Display in the main UI is correct though.

**Reply by WILLYWZ** [reply]
it's deafult fan curve and fan curve was not edited. only fan calibration is done.

**Reply by WILLYWZ** [reply]
Also minor details. 
minutes till hibernation in sleep on battery can be translated as 离电睡眠模式下，几分钟后进入休眠模式

**Reply by WILLYWZ** [reply]
enable gpu on shutdown can be 在关机时启用GPU

**Reply by seerge** [reply]
@WILLYWZ 

To answer your remarks: 

- "Default" fan curves that you see on chart - is what BIOS "reports" as being default. But in practice it may not be always reflecting actual default fan behavior
- When you set a "custom" fan curves, firmware / BIOS not always follows them precisely. It's absolutely normal.
- Custom fan curve is set in some abstract % points. Where BIOS interprets 0% as OFF, but 1% is immediately something like 1600-2000RPM. At the same time actual fan speed is reported in RPM. So it will be always some mismatch here. Fan calibration (measuring how much RPM is 100%) helps a bit.

So to summarize: don't expect neither default nor custom fan curve to be followed by firmware/BIOS precisely. Treat them rather as a guidance and just enjoy your laptop :) 

----

Translations are managed via [Crowdin platform](https://crowdin.com/project/g-helper). If something is not very accurate there - you can offer translations there.

Thanks

**Reply by seerge** [reply]
@WILLYWZ i will keep it open:) some explanations here could be handy addition to existing FAQ

**Reply by WILLYWZ** [reply]
understand. Thanks.
